README.md

```diff
@@ -798,6 +798,8 @@ iobroker object set alias.0.aliasName common.alias.read="read-func"
 iobroker object set alias.0.aliasName common.alias.write="write-func"
 ```
 
+> **_NOTE:_** The permissions of the source object are ignored and only the permissions set on the alias object itself are relevant for database operations. 
```

Comment:
du kennst [https://github.com/orgs/community/discussions/16925 ? Sollte gehen](https://docs.github.com/de/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)

Reply:
Jedenfalls dann lokal unhübsch mmn oben die Syntax aus dem Example unten das aktuelle. 

![grafik](https://github.com/ioBroker/ioBroker.js-controller/assets/25484431/81e88207-809c-4042-bae5-b0f9f7471d7f)